pkg/lease/lease_controller.go

```diff
@@ -75,21 +83,42 @@ func (r *leaseUpdater) reconcile(ctx context.Context) {
 				},
 			},
 		}
-		if _, err := r.kubeClient.CoordinationV1().Leases(r.leaseNamespace).Create(context.TODO(), lease, metav1.CreateOptions{}); err != nil {
-			klog.Errorf("unable to create addon lease %q/%q on hub cluster. error:%v", r.leaseNamespace, r.leaseName, err)
+		if _, err := client.CoordinationV1().Leases(r.leaseNamespace).Create(ctx, lease, metav1.CreateOptions{}); err != nil {
```

Comment:
Should we use r.leaseNamespace or namespace here?

Reply:
updated